recipes-core/everest/everest-core_2024.6.0.bb

```diff
@@ -30,19 +30,26 @@ DEPENDS = " \
     libslac \
     libevent \
     libevse-security \
+    libcbv2g \
     mbedtls \
-    openv2g \
     curl \
     sqlitecpp \
 "
 
-RDEPENDS:${PN} += "openv2g libevent mbedtls"
+RDEPENDS:${PN} += "libevent mbedtls"
 
 INSANE_SKIP:${PN} = "already-stripped useless-rpaths arch file-rdeps"
 
 FILES:${PN} += "${datadir}/everest/*"
 
-EXTRA_OECMAKE += "-DDISABLE_EDM=ON -DNO_FETCH_CONTENT=ON -DEVEREST_ENABLE_RUN_SCRIPT_GENERATION=OFF"
+EXTRA_OECMAKE += " \
+    -DDISABLE_EDM=ON \
+    -DNO_FETCH_CONTENT=ON \
+    -DEVEREST_ENABLE_RUN_SCRIPT_GENERATION=OFF \
+    -Deverest-core_INSTALL_EV_CLI_IN_PYTHON_VENV=OFF \
+    -Deverest-core_USE_PYTHON_VENV=OFF \
+    -DEV_SETUP_PYTHON_EXECUTABLE_USE_PYTHON_VENV=OFF \
```

Comment:
My problem with this is that we have to add these three flags, with the respective PROJECT_NAME, to all recipes that build additional EVerest modules. I can not think of a solution right now for this.

Reply:
I think that's a totally valid solution, not sure we can come up with another one. However we already have to add a few flags anyway at the moment, so this just adds a few more flags:
```
    -DDISABLE_EDM=ON \
    -DNO_FETCH_CONTENT=ON \
    -DEVEREST_ENABLE_RUN_SCRIPT_GENERATION=OFF \
    -DEVEREST_PROJECT_DIRS:STRING=${STAGING_DIR_TARGET}/usr/share/everest/ \
```

Reply:
If you have a lot of these recipes: An alternative for you could be to patch-out the lines referring to the venv in a everest-core bbappend:
https://github.com/EVerest/everest-core/blob/4b13b10c4b6ba7ae3edf0a9f7ddfdce274a9332a/cmake/everest-generate.cmake#L132
https://github.com/EVerest/everest-core/blob/4b13b10c4b6ba7ae3edf0a9f7ddfdce274a9332a/cmake/ev-cli.cmake#L9

And add a 
find_package(Python3
            REQUIRED
            COMPONENTS
                Interpreter
        )
If needed

Reply:
Yes I already did this before, but right now we just added the flags in the recipes so that it builds.

Reply:
Maybe we can find an easier solution for the next release in everest-core, nothing immediately comes to mind but I'll think about it

Reply:
It's also quite possible that setting
```
-Dyour_project_name_USE_PYTHON_VENV=OFF
```
Is actually enough since there were some fixes related to this

Reply:
Yes. Just tested it and at least I do not get any configure issues.